stdlib/contrib/mhall119/helpers/helpers.flux

```diff
@@ -0,0 +1,94 @@
+// Package helpers provides shortcut functions for small common tasks
+//
+// ## Metadata
+// introduced: ??
+package helpers
+
+import "array"
+import "json"
+// yieldValue lets you display a scalar in the InfluxDB UI by
+// wrapping it in an Array
+//
+// ## Parameters
+// - v: Input data.
+// - name: Name to pass to the subsequent yield() call.  Default is 'value'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldValue(v: testData.testValue)
+// ```
```

Comment:
~~~suggestion
// ```
// import "contrib/mhall119/helpers"
//
// option testData = {testValue: 125}
// option now = () => 2022-01-01T00:00:00Z
//
// > helpers.yieldValue(v: testData.testValue)
// ```
~~~

Reply:
What does adding `now` here change? As far as I can tell it's not used anywhere.

Reply:
Your output stream includes a `_time` column that's populated with `now()`. Setting this option will make the output consistent. If you don't set it, `now()` will return the time that the docs are generated. We can and should show the output from this functions, which is why I added the `>`. This appends a yield named "output" at the end of that line.

This also goes to Nathaniel's point using implicit yields. As-is, this example will actually output two yields. One named "value" and the other, "output".

Reply:
Something else to note is that if you don't want the `option now` line to _display_ as part of the example, you can add a `#` at the beginning of the line. For example:

```javascript
// # option now = () => 2022-01-01T00:00:00Z
```

When we display the example, that line won't be included. But it would still be used when generating the output from the example.